content/api/servo-assembly/arduino-commands.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Custom Arduino Commands
+---
+
+At the core of the servo assembly is an [Arduino Uno][arduino]. By default the
+Arduino runs firmware created by SourceBots providing simple access to the
+attached servo shield, compatible ultrasound sensors and to the GPIO pins
+directly.
+
+You can also extend the firmware on the Arduino to support more features, for
+example if you have other types of sensor which require higher precision timing
+than can be achieved through the Python API.
+
+## Requirements
+
+As your new firmware will still need to be able to talk to the Python code
+running on the Raspberry Pi, you'll need to base it on SourceBots default
+firmware. You can get a copy of the source code from GitHub either by
+downloading the [repo archive](servo-firmware-zip) or by cloning the repo:
```

Comment:
Parentheses -> square brackets

Currently this is resulting in a broken link to [https://HOSTNAME/api/servo-assembly/arduino-commands/servo-firmware-zip](https://deploy-preview-87--sourcebots-docs.netlify.com/api/servo-assembly/arduino-commands/servo-firmware-zip).

Reply:
good spot: 422a0f8

Reply:
Is there a way we can test that all internal links go somewhere?

Reply:
I'm sure a broken link checker (e.g. [broken-link-checker-local](https://www.npmjs.com/package/broken-link-checker-local)) would be a useful addition to the CI tests.

Reply:
#89